CHANGELOG.md

```diff
@@ -16,9 +18,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - [2115](https://github.com/FuelLabs/fuel-core/pull/2115): Add test for `SignMode` `is_available` method.
 - [2124](https://github.com/FuelLabs/fuel-core/pull/2124): Generalize the way p2p req/res protocol handles requests.
 
-### Added
-- [2119](https://github.com/FuelLabs/fuel-core/pull/2119): GraphQL query fields for retrieving information about upgrades.
-- [2116](https://github.com/FuelLabs/fuel-core/pull/2116): Replace `H160` in config and cli options of relayer by `Bytes20` of `fuel-types`
+### Fixed
+- [2134](https://github.com/FuelLabs/fuel-core/pull/2134): Perform RecoveryID normalization for AWS KMS -generated signatures.
```

Comment:
```suggestion
- [2134](https://github.com/FuelLabs/fuel-core/pull/2134): Perform RecoveryID normalization for AWS KMS generated signatures.
```

Reply:
Oh good point I ddn't realize the space-before-dash compound-noun-modifiers are nonstandard English. After a long consultation with ChatGPT and couple of style guides, I'm still convinced that the form I used clearner than the alternative without the dash. If you feel strongly about this, the sentence could be reformulated as 

> Perform RecoveryID normalization for signatures generated by AWS KMS.